src/cameras.jl

```diff
@@ -55,7 +67,9 @@ function camera_set!(rendermodel::RenderModel, cam::CarFollowCamera{I}, scene::E
 
     veh_index = findfirst(cam.targetid, scene)
     if veh_index != nothing
-        camera_set_pos!(rendermodel, scene[veh_index].state.posG)
+        target_pos = scene[veh_index].state.posG
+        camera_pos = VecSE2(cam.follow_x ? target_pos.x : 0., cam.follow_y ? target_pos.y : 0.)
```

Comment:
should the default position to 0.0 be a parameter? 
(it should be documented at least)